.github/actions/setup-environment/action.yml

```diff
@@ -10,6 +10,8 @@ inputs:
     type: bool
   eve_image:
     type: string
+  eve_cache:
+    type: string
```

Comment:
"eve_cache"? It could get confusing, people might think it is a cache location for lots of things, or maybe something to do with eve itself? Would it be ok to use something more explicit, like `eve_image_file` or `eve_image_tar` or something like that?

Reply:
renamed to eve_image_cache_key